portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessDeliveryStructuredContent.testcase

```diff
@@ -65,4 +65,62 @@ definition {
 		}
 	}
 
+	@description = "StructuredContent can be filtered with a set priority"
+	@priority = "5"
+	test StructuredContentContainsUpdatedPriorityValueFilteredByEqualTo {
+		property portal.acceptance = "true";
+
+		task ("Given three web contents are created with default priority") {
+			var ddmStructureId = WebContentStructures.getDdmStructureId(structureName = "content-structure");
+
+			var responseToParse1 = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title 1");
+			var responseToParse2 = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title 2");
+			var responseToParse3 = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title 3");
+		}
+
+		task ("Given edit a web content with different priorities") {
+			var editStructuredContentId = HeadlessWebcontentAPI.editStructuredContentWithPriority(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				priority = "1.3",
+				responseToParse = "${responseToParse2}",
+				title = "WC WebContent Title 2");
+		}
+
+		task ("When web contents are filtered with 1.3 priority") {
+			var response = HeadlessWebcontentAPI.filterStructuredContentWithPriority(
+				editStructuredContentId = "${editStructuredContentId}",
+				filtervalue = "priority%20eq%201.3");
+		}
+
+		task ("Then the response body includes the 1.3 priority field") {
+			HeadlessWebcontentAPI.assertPriorityFieldIsFilterWithDefaultValue(
+				expectedPriorityDefaultValue = "1.3",
+				responseToParse = "${response}");
+		}
+
+		task ("Then assert only one element has filtered") {
+			HeadlessWebcontentAPI.assertOnlyOneElementHasFiltered(
```

Comment:
Since we check the same response we can do both asserts in the step `"Then the response body includes the 1.3 priority field"` (we don't perform any extra action in order to check the count). But maybe we can modify a bit the step itself and change it to a more accurate one, like `"Then the response body only includes items with priority 1.3"`. What do you think?

Reply:
you are right, I will edit.